sentry-core/src/api.rs

```diff
@@ -167,9 +148,5 @@ where
 
 /// Returns the last event ID captured.
 pub fn last_event_id() -> Option<internals::Uuid> {
```

Comment:
_nit_: Unrelated to this change, could you clarify in the docs that this returns the last event id from the current execution context, i.e. the active Hub?